meson.build

```diff
@@ -63,6 +63,8 @@ csp_inc = include_directories(['.', 'include'])
 install_subdir('include', install_dir : '.')
 install_headers(csp_config_h, install_dir : 'include/csp')
 
+csp_deps += dependency('freertos', fallback: ['freertos', 'freertos_dep'], required: false)
```

Comment:
Why add this here, the FreeRTOS dep is included by the meson build file in sec/arch/FreeRTOS/meson build

Reply:
I couldn't get it to compile otherwise. Try compiling on C21.

Reply:
We do compile for c21 all the time. Did you remember to set the OS to FreeRTOS? So the right meson.buils include file is included?